src/Provider/UnitTests/Provider.Web/UtilityTests/CheckRouteIsValidForVacancyTests.cs

```diff
@@ -7,7 +7,7 @@
 using FluentAssertions;
 using Xunit;
 
-namespace Esfa.Recruit.UnitTests.Provider.Web.UtilityTests
+namespace Esfa.Recruit.Provider.UnitTests.Provider.Web.UtilityTests
```

Comment:
Why do we have to have Provider twice. The namespace should be `Esfa.Recruit.UnitTests.Provider.Web.UtilityTests`

This is not consistent but this is what we need to start doing, have UnitTests at the higher level in the namespace and component breakdown after.